examples/raytrace-parallel/Cargo.toml

```diff
@@ -11,6 +11,8 @@ crate-type = ["cdylib"]
 console_error_panic_hook = "0.1"
 futures = "0.1"
 js-sys = "0.3.22"
+rayon = "1.0.3"
+rayon-core = "1.0.3"
```

Comment:
I know you're patching to git anyway, but `spawn_handler` will need rayon-1.1 / rayon-core-1.5.
(I'm working on those release notes now...)

Reply:
Oh nice! We'll just wait for those in that case :)

Reply:
These are now published!